CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+### 1.6.1
+- Improve naming of widget so it's more distinguishable.
+- Bugfix - validate credentials fails if password contains '&' character.
```

Comment:
Please add a link to an issue or PR to each change.

And please add an empty line before `### 1.6.0`

Reply:
This got resolved as I posted it. Nevermind.